unit-test/background/tracker-allowlist-tests.js

```diff
@@ -1,10 +1,30 @@
+const tdsStorageStub = require('./../helpers/tds.es6')
+const tds = require('../../shared/js/background/trackers.es6')
+const tdsStorage = require('../../shared/js/background/storage/tds.es6')
+
 const refTests = require('./reference-tests/tracker-radar-tests/TR-domain-matching/tracker_allowlist_matching_tests.json')
 const allowList = require('./../..//shared/js/background/allowlisted-trackers.es6')
+const config = require('./../data/extension-config.json')
+
+describe('Tracker allowlist tests:', () => {
+    beforeAll(() => {
```

Comment:
this is an old set of tests but it started causing problems because it didn't have `beforeAll` section setting up the custom config (instead custom config was hardcoded elsewhere)